lib/ts/recipe/emailverification/recipe.tsx

```diff
@@ -55,7 +56,13 @@ export default class EmailVerification extends RecipeModule<
         super(normaliseEmailVerificationFeature(config));
 
         {
-            const builder = new OverrideableBuilder(RecipeImplementation(this.config.recipeId, this.config.appInfo));
+            const webJsImplementation = new WebJSEmailVerification({
+                appInfo: config.appInfo,
+                recipeId: config.recipeId,
+                preAPIHook: config.preAPIHook,
```

Comment:
Add postAPI hook too

Reply:
Done, added postAPIHook to the config for the recipe module (used by all recipes)